lib/src/coap_request.dart

```diff
@@ -86,13 +86,13 @@ class CoapRequest extends CoapMessage {
     _uri = value;
   }
 
-  CoapIEndPoint? _endpoint;
+  CoapEndPoint? _endpoint;
```

Comment:
Could perhaps be renamed `CoapEndpoint` or even `Endpoint` (endpoint is one word but capitalized as two in `CoapEndPoint`).

Reply:
Should be addressed now :) I renamed a couple of other files/classes as well. There are still some left which could be addressed in a follow-up PR, though.